### ISSUE TITLE
document initImmediate

### ISSUE THREAD

**Issue by arve0** [i18next/i18next#828]
Is this option documented? A [google search](https://www.google.no/search?client=safari&rls=en&q=initImmediate+site:i18next.com&ie=UTF-8&oe=UTF-8&gfe_rd=cr&ei=bVgIWNffIbKr8wedo6FQ) for it on i18next.com was unable to find it.

ref #650

**Comment by jamuhl** [comment]
http://i18next.com/docs/options/#init-options

documented: yes
easy to understand: probably not really
accepting PR: yes

;)

**Comment by arve0** [comment]
Ah, sorry for the noise. Agree on easy to understand note. Not because of bad documentation but the name `initImmediate`. Connected *immediate* to the English meaning, happening at once, not the JS meaning, async, that `setImmediate` gives (funny I spelled the title wrong here and still not connected it).

I feel the name `initAsync` would give more sense (not caring about the specific implementation), but not that bad to break API. Closing.
